src/components/Tree/index.jsx

```diff
@@ -0,0 +1,193 @@
+import { Component } from 'react';
+import { arrayOf, shape, string, func, oneOfType, object } from 'prop-types';
+import { memoizeWith } from 'ramda';
+import { withStyles } from 'material-ui/styles';
+import { ListItem } from 'material-ui/List';
+import ExpansionPanel, {
+  ExpansionPanelSummary,
+  ExpansionPanelDetails,
+} from 'material-ui/ExpansionPanel';
+import ChevronDownIcon from 'mdi-react/ChevronDownIcon';
+import theme from '../../theme';
```

Comment:
If you need the theme in the component, pass `withTheme: true` to the options and it will become a prop.

```js
@withStyles(
  theme => ({
    // styles...
  }),
  {
    // options
    withTheme: true,
  }
)
export default class Tree extends Component {
  method() {
    this.props.theme
  }
}
```